### PR TITLE
JDG-1773 Red Hat Cache Service

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/JDG-1885

Hey!

According to the [How to get things into Online](https://docs.google.com/document/d/1jc0EqfbcX6584taNdvSm9wio64hh6N46oVj5yUlbLoc/edit#heading=h.60ys2afk1e1a) document we would like to introduce Red Hat Cache Service to the Pro tier. 

I'm not entirely sure what is the proper workflow for the Pull Request. As the document states, the nightly job replaces the templates with fresh ones. Shall I just delete everything apart from `official.yaml` entries?

//cc @tristantarrant @ryanemerson @osnipso @sspeiche @luciddreamz

### CODE REVIEW

File: official.yaml [hunk 1]
```diff
@@ -252,6 +252,8 @@ data:
           - online-professional
   datagrid:
     templates:
+      - location: https://raw.githubusercontent.com/jboss-container-images/datagrid-7-image/datagrid-services-dev/templates/caching-service.json
+        docs: https://github.com/jboss-container-images/datagrid-7-image/blob/datagrid-services-dev/documentation/cache-service.asciidoc
```
Comment (bparees) [inline official.yaml]: if you want this in pro you need to add the appropriate tags.

**Comment by bparees** [comment]
> As the document states, the nightly job replaces the templates with fresh ones. 

The nightly job does a fresh import of everything from official/community.yaml.

> Shall I just delete everything apart from official.yaml entries?

are you trying to remove something?

**Comment by slaskawi** [comment]
@bparees Ok, I think I understand now. Travis failure gave me a hint. Let me update this PR.

File: official.yaml (hunk 1 above)
Comment (slaskawi) [reply official.yaml]: Now it should be fine. We would like to make it available in the `Pro` tier.

**Comment by slaskawi** [comment]
@bparees Updated.

**Comment by sspeiche** [comment]
+1

**Comment by slaskawi** [comment]
@sspeiche @bparees Thank you!

//cc @tristantarrant @ryanemerson

**Pull request merged**